src/page-weekly-snapshot/LocaleSummaryTable.tsx

```diff
@@ -0,0 +1,368 @@
+  let result = 0;
+  let hasData = false;
+  const valueKeys = get(VALUE_MAPPING, value);
+  for (let i = 0; i < facilities.length; i++) {
+    const modelInputs = facilities[i].modelInputs;
+    const data = omit(
+      pick(modelInputs, valueKeys),
+      "staffDeaths",
+      "staffRecovered",
+      "staffCases",
+    );
+    if (keys(data).length > 0) {
+      hasData = true;
+    }
+    result += sum(values(data));
+  }
+  const incarceratedData: incarceratedData = {
+    incarceratedData: result,
+    hasData: hasData,
+  };
+  return incarceratedData;
+}
+
+function makeIncarceratedDataRow(
+  hasData: boolean,
+  incarceratedDataPerCapita: number,
+) {
+  if (hasData) {
+    return (
+      <tr>
+        <TextContainer>
+          <Left>{formatThousands(incarceratedDataPerCapita)}</Left>
+        </TextContainer>
+      </tr>
+    );
+  } else {
+    return (
+      <tr>
+        <TextContainer>
+          <Left>???</Left>
+        </TextContainer>
+      </tr>
+    );
+  }
+}
+
+function makeTableColumn(
+  heading: string,
+  incarceratedData: number,
+  stateData: number,
+  stateRank: number,
+  hasData: boolean,
+) {
+  const incarceratedDataRow = makeIncarceratedDataRow(
+    hasData,
+    incarceratedData,
+  );
+  return (
+    <>
+      <tr>
+        <TableHeadingCell>
+          <TextContainerHeading>
+            <Right>{heading} </Right>
+            <LeftHeading>(per 100k)</LeftHeading>
+          </TextContainerHeading>
+        </TableHeadingCell>
+      </tr>
+      <BorderDiv />
+      <tr>
+        <TableCell>
+          Incarcerated {heading}
+          <HorizontalRule />
+        </TableCell>
+      </tr>
+      {incarceratedDataRow}
+      <BorderDiv />
+      <tr>
+        <TableCell>
+          Overall State {heading}
+          <HorizontalRule />
+        </TableCell>
+      </tr>
+      <tr>
+        <TableCell>
+          <TextContainer>
+            <Left>{formatThousands(stateData)}</Left>
+            <Right>
+              {formatOrdinal(stateRank)} lowest of {NUM_STATES}
+            </Right>
+          </TextContainer>
+        </TableCell>
+      </tr>
+    </>
+  );
+}
+
+const LocaleSummaryTable: React.FC<{
+  stateName: string | undefined;
+  stateNames: string[];
+}> = ({ stateName, stateNames }) => {
+  const localeState = useLocaleDataState();
+  const localeData = localeState.data;
+  const {
+    state: { facilities: facilitiesState },
+  } = useFacilities();
+  const facilities = Object.values(facilitiesState);
+
+  let casesPerCapita = 0;
+  let casesPerCapitaRank = 0;
+  let deathsPerCapita = 0;
+  let deathsPerCapitaRank = 0;
+
+  let incarceratedCasesPerCapita = 0;
+  let incarceratedDeathsPerCapita = 0;
+
+  let hasCaseData = true;
+  let hasDeathData = true;
+
+  const allStateMetrics = getAllStateData(localeData, stateNames);
+
+  // TODO: currently getting state name from the drop down
+  // will need to update whenever that changes
```

Comment:
if this isn't a TODO for this PR it should have a ticket number associated with it